.github/workflows/integration-linux.yml

```diff
@@ -12,7 +12,7 @@ concurrency:
 jobs:
 
   build:
-    name: PR integration tests (linux)
+    name: PR integration tests (linux) 
```

Comment:
nit: I don't think this change is necessary

Reply:
... I remember I reverted this.. doing it now